### PR TITLE
fix(addon-editor): fix tab behaviour

### LINKED ISSUE #2334: 🐞 - Не удаляется табуляция в таблице
### Which @taiga-ui/* package(s) are the source of the bug?

addon-editor

### Please provide a link to a minimal reproduction of the bug

https://taiga-ui.dev/components/editor-new

### Is this issue blocking you?

Non-Blocking

### Description

1. Создать таблицу в редакторе
2. В ячейке таблицы добавить табуляцию
3. Попытаться удалить backspace'ом

### Angular version

11.2.11

### Taiga UI version

2.55.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #2334 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/addon-editor/extensions/indent-outdent.ts [hunk 1]
```diff
@@ -1,217 +1,19 @@
...
         return {
             Tab: () =>
                 tuiIsOrderedOrBulletList(this.editor)
                     ? null
-                    : this.editor.commands.indent(),
-            'Shift-Tab': () =>
-                tuiIsOrderedOrBulletList(this.editor)
-                    ? null
-                    : this.editor.commands.indent(),
+                    : this.editor.commands.insertContent(`\t`),
```
Comment (waterplea) [inline projects/addon-editor/extensions/indent-outdent.ts:16]: Does this mean all editor and sockets have to have `white-space: pre-wrap`?

File: projects/addon-editor/extensions/indent-outdent.ts [hunk 2]
```diff
@@ -1,217 +1,19 @@
...
-        };
-    },
-
     addKeyboardShortcuts(): any {
         return {
             Tab: () =>
                 tuiIsOrderedOrBulletList(this.editor)
                     ? null
-                    : this.editor.commands.indent(),
-            'Shift-Tab': () =>
```
Comment (waterplea) [inline projects/addon-editor/extensions/indent-outdent.ts:211]: What about removing \t symbols?

File: projects/addon-editor/extensions/indent-outdent.ts (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/addon-editor/extensions/indent-outdent.ts:16]: Editor already has `white-space: pre-wrap`. And user can add it to socket like in example 4, if he want to display multiple spaces/tab. Or do you think we should add it to socket by default ?

File: projects/addon-editor/extensions/indent-outdent.ts (hunk 2 above)
Comment (vladimirpotekhin) [reply projects/addon-editor/extensions/indent-outdent.ts:211]: just backspace

File: projects/addon-editor/extensions/indent-outdent.ts (hunk 1 above)
Comment (splincode) [reply projects/addon-editor/extensions/indent-outdent.ts:16]: `Or do you think we should add it to socket by default ` it's good point I think so

**Review by waterplea — APPROVED** [review]

**Pull request merged**